tox.ini

```diff
@@ -4,7 +4,7 @@ envlist = lint,py3
 [testenv]
 passenv = CI TRAVIS TRAVIS_*
 commands =
-    coverage run -m pytest
+    coverage run -m pytest {posargs}
```

Comment:
Just so I understand - What’s this allowed you to achieve?

Reply:
This allows us to pass positional arguments along to pytest through tox:
```bash
$ tox -e py3 -- -k foo
```
Unless I'm missing something this isn't currently possible?